### PR TITLE
New Port Belle Boomerang

### PR DESCRIPTION
**Pull request by Fraxinus88**
# New Port for Belle Boomerang

## Game Information
- **Title**: Belle Boomerang
- **URL**: https://store.steampowered.com/app/1632170/Belle_Boomerang/

## CFW Tests:
[X] AmberELEC
[X] ArkOS
[X] JELOS/ROCKNIX
[X] MuOS

Architecture
[X] aarch64
[X] Panfrost (Mainline)

Resolution Tests
Test all major resolutions:
[ ] 480x320
[X] 640x480
[X] 720x720 (RGB30)
[X] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - gameinfo.xml
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

File: ports/belleboomerang/screenshot.png
Comment (cdeletre) [inline ports/belleboomerang/screenshot.png:1]: File size can be reduced
Use pngquant or an online tool :)

File: ports/belleboomerang/cover.png
Comment (cdeletre) [inline ports/belleboomerang/cover.png:1]: File size can be reduced.
Use pngquant or an online tool :)
 Also cover resolution can be lowered to 400x300

Comment (Fraxinus88) [reply ports/belleboomerang/cover.png:1]: Okies, have shrunk the file sizes :) Can I keep it as 640 x 480?

File: ports/belleboomerang/screenshot.png
Comment (Fraxinus88) [reply ports/belleboomerang/screenshot.png:1]: Okies!

File: ports/belleboomerang/cover.png
Comment (cdeletre) [reply ports/belleboomerang/cover.png:1]: 640x480 is fine

**Review by cdeletre — APPROVED** [review]

**Review by Cebion — APPROVED** [review]

**Pull request merged**